internal/pkg/cli/deploy/workload.go

```diff
@@ -311,40 +345,52 @@ func (d *workloadDeployer) uploadContainerImage(imgBuilderPusher imageBuilderPus
 		return nil, nil
 	}
 	// If it is built from local Dockerfile, build and push to the ECR repo.
-	buildArg, err := buildArgs(d.name, d.imageTag, d.workspacePath, d.mft)
+	buildArgsPerContainer, err := buildArgsPerContainer(d.name, d.workspacePath, d.image, d.mft)
```

Comment:
dumb question: is `buildArgsPerContainer` is just building docker args for the main container for now?

Reply:
yes. currently we have only build args for main container, in the next PR I will add the sidecar build args as well.